README.md

```diff
@@ -40,7 +52,7 @@ sensitivity analyses and questionnaires.
 
 ## References and Documentation
 
-* Please refer to the [`{admiral}` References and Documentation](https://pharmaverse.github.io/admiral/index.html#references-and-documentation)
+* Please refer to the [{admiral} References and Documentation](https://pharmaverse.github.io/admiral/main/index.html#references-and-documentation)
```

Comment:
Should it be `cran-release` instead of `main`?

Reply:
@cicdguy should we change this one also ? (If yes I will also update the .Rmd admiral_homepage but just let me know !)

Reply:
Okay I am just fixing it :)